app/controllers/split_checkout_controller.rb

```diff
@@ -128,4 +128,14 @@ def redirect_to_step
       redirect_to order_path(@order)
     end
   end
+
+  def redirect_to_step
```

Comment:
The class is growing to big. This method is good in the controller though. Shall we pull some of the checkout logic out into a service?

Reply:
Tech-debt issue opened: https://github.com/openfoodfoundation/openfoodnetwork/issues/8942